src/types.php

```diff
@@ -308,6 +308,28 @@ public static function iterable(Type $keyType = self::mixed, Type $valueType = s
         return new IterableType($keyType, $valueType);
     }
 
+    /**
+     * @psalm-pure
+     * @param array<string, Property> $properties
```

Comment:
Should be `@param array<string, Type|Property> $properties`

Reply:
Thanks, fixed!